client/app/states/vms/details/details.html

```diff
@@ -0,0 +1,513 @@
+<ol class="breadcrumb">
+  <li>
+    <a ui-sref="services">{{'My Services'|translate}}</a>
+  </li>
+  <li>
+    <a ui-sref="services.details({serviceId: vm.vmDetails.service.id})">{{ ::vm.vmDetails.service.name }}</a>
+  </li>
+  <li>
+    <a ui-sref="vms.details({vmId: vm.vmDetails.id})">{{ ::vm.vmDetails.name }}</a>
+  </li>
+</ol>
+<div class="ss-details-wrapper">
+	<table class="vm-details-table">
+		<tr>
+			<th colspan="2">
+				{{'Properties'|translate}}
+			</th>
+		</tr>
+		<tr>
+			<td>{{'Name'|translate}}</td>
+			<td>{{vm.vmDetails.name}}</td>
+		</tr>
+		<tr>
+			<td>{{'Hostnames'|translate}}</td>
+			<td>{{vm.vmDetails.hostnames.join(",")}}</td>
+		</tr>
+		<tr>
+			<td>{{'IP Addresses'|translate}}</td>
+			<td>{{vm.vmDetails.ipaddresses.join(",")}}</td>
+		</tr>
+		<tr ng-show="vm.vmDetails.instance">
+			<td>{{'MAC Address'|translate}}</td>
+			<td>{{vm.vmDetails.instance.mac_addresses.join(",")}}</td>
+		</tr>
+		<tr>
+			<td>
+				{{'Container'|translate}}
+			</td>
+			<td>
+				{{vm.vmDetails.vendor+ ': '+vm.vmDetails.hardware.cpu_total_cores+ ' CPUs ('+vm.vmDetails.hardware.cpu_sockets+' sockets
+				x ' + vm.vmDetails.hardware.cpu_cores_per_socket +' core), '+vm.vmDetails.hardware.memory_mb + ' MB'}}
+			</td>
+		</tr>
+		<tr ng-show="vm.vmDetails.host">
+			<td>{{'Parent Host'|translate}} / {{'Node Platform'|translate}}</td>
+			<td>{{vm.vmDetails.host.vmm_product}}</td>
+		</tr>
+		<tr>
+			<td>{{'Platform Tools'|translate}}</td>
+			<td>{{vm.vmDetails.tools_status}}</td>
+		</tr>
+		<tr>
+			<td>
+				{{'Operating System'|translate}}
+			</td>
+			<td>
+				{{vm.vmDetails.hardware.guest_os_full_name}}
+			</td>
+		</tr>
+
+		<tr ng-show="vm.vmDetails.hardware.bitness">
+			<td>
+				{{'Architecture'|translate}}
+			</td>
+			<td>
+				{{vm.vmDetails.hardware.bitness}} bit
+			</td>
+		</tr>
+		<tr ng-show="vm.vmDetails.cloud===false">
+			<td>
+				{{'Devices'|translate}}
+			</td>
+			<td>
+				<i class="fa fa-hdd-o fa-lg" aria-hidden="true"></i>{{3 + vm.vmDetails.num_disks}}
+			</td>
+		</tr>
+		<tr ng-show="vm.vmDetails.cloud===false">
+			<td>
+				{{'CPU Affinity'|translate}}
+			</td>
+			<td>
+				{{vm.vmDetails.cpu_affinity}}
+			</td>
+		</tr>
+		<tr>
+			<td>
+				{{'Snapshots'|translate}}
+			</td>
+			<td>
+				<i class="fa fa-camera fa-lg" aria-hidden="true"></i>{{vm.vmDetails.snapshotCount}}
+			</td>
+		</tr>
+		<tr>
+			<td>
+				{{'Advanced Settings'|translate}}
+			</td>
+			<td>
+				<i class="fa fa-wrench fa-lg" aria-hidden="true"></i>{{vm.vmDetails.advanced_settings.length}}
```

Comment:
we don't (presently) but should (some day) use aria, no reason to specify aria-hidden here